docs/docs/workflows/config.md

```diff
@@ -231,13 +231,13 @@ The following describes the configuration fields and how you obtain the required
 
 * **iamProfileControlPlane**: The name of an IAM instance profile attached to all control-plane nodes.
 
-  Use the [provided Terraform script](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam) to generate the necessary profile. The profile name will be provided as Terraform output value: `control_plane_instance_profile`.
+  The resource can be created with [Terraform](../reference/terraform.md). For that, use the [provided Terraform script](https://github.com/edgelesssys/constellation/tree/release/v2.2/hack/terraform/aws/iam) to generate the necessary profile. The profile name will be provided as Terraform output value: `control_plane_instance_profile`.
```

Comment:
I think the reference could be confusing. While the new page introduced Terraform, it does not mention the generation of IAM instance profiles. Profile generation is also not supported (hack), keeping it seperated from official features will help to set expectations.